### PR TITLE
[MRG] update members

### PR DESCRIPTION
**Pull request by ctb**
please someone review ;)

### CODE REVIEW

File: members.rst [hunk 1]
```diff
@@ -9,22 +9,22 @@ Current members
 =======================  ============================   =================================
 Name                     Position                       More information
 =======================  ============================   =================================
+Mo Abuelanin             Graduate student
 \C. Titus Brown          QSWG and Assoc Professor       http://ivory.idyll.org/
```
Comment (mr-eyes) [inline members.rst:13]: Mohamed Abuelanin, Graduate Student, https://mr-eyes.github.io

File: members.rst [hunk 2]
```diff
@@ -40,13 +40,19 @@ Sherine Awad             https://github.com/SherineAwad
 Phillip Brooks           https://brooksph.github.io
 Michael R. Crusoe        https://orcid.org/0000-0002-2961-9670
 Jake Fenton              https://github.com/bocajnotnef
```
Comment (taylorreiter) [inline members.rst:42]:
```suggestion
Jake Fenton              https://github.com/bocajnotnef
Alicia Gingrich
```

**Review by taylorreiter — APPROVED** [review]
added alicia as alum, otherwise i think LGTM?

**Comment by ctb** [comment]
can you edit the pull request, please?

On Sat, May 01, 2021 at 11:01:16AM -0700, Mohamed Abuelanin wrote:
> @mr-eyes commented on this pull request.
> 
> 
> 
> > @@ -9,22 +9,22 @@ Current members
>  =======================  ============================   =================================
>  Name                     Position                       More information
>  =======================  ============================   =================================
> +Mo Abuelanin             Graduate student
>  \C. Titus Brown          QSWG and Assoc Professor       http://ivory.idyll.org/
> 
> Graduate Student, https://mr-eyes.github.io
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/dib-lab/dib-lab/pull/97#pullrequestreview-649806256
-- 
C. Titus Brown, ***@***.***

**Review by ACharbonneau — APPROVED** [review]

**Pull request merged**